src/CalcManager/UnitConverter.h

```diff
@@ -119,9 +119,9 @@ namespace UnitConversionManager
     class CategoryHash
```

Comment:
We should just get rid of CategoryHash completely. The unordered_map can use the default std::hash<int> now.

Reply:
great observation, @mcooley pushed updated changes for the comments.